hardware/asic/README.md

```diff
@@ -327,7 +331,7 @@ As the skywater tech files are not installed in magic’s library, we need to cr
 
 So enter following for the symbolic links:
 ```bash
-sudo ln -s <sky130A_install_root_dir>/sky130A/libs.tech/magic/*../usr/local/lib/magic/sys/
+sudo ln -s <sky130A_install_root_dir>/sky130A/libs.tech/magic/    /usr/local/lib/magic/sys/
```

Comment:
I did not need this 

One of the things I installed automatically copied the files to /usr/local/lib/magic/sys/

You may have missed some "make install" step

Reply:
Sir I followed the following approach
As the skywater tech files are not installed in magic’s library, we need to create a symbolic link in order to use the tech files for drawing layout.

Reply:
You are right but I think the * I needed 
sudo ln -s <sky130A_install_root_dir>/sky130A/libs.tech/magic/*   /usr/local/lib/magic/sys/

Reply:
ok I will add in the next push plz merge this and i will push another PR also with PDK_ROOT removed